calculator.js

```diff
@@ -1,28 +1,58 @@
-
-const exampleAdditionInput = {
+const inputSample = {
   num1: 3,
-  num2: 5,
-  operation: 'add',
+  num2: 6,
+  operation: '+',
 }
-// '3 + 5 = 8'
 
-const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+const calculateUserInput = function (error, input) {
+  const operation = input.operation;
+
+  if (isNaN(input.num1)) {
+    console.log(`${input.num1} is not a number`)
+    return
+  }
+  const num_1 = Number(input.num1)
+
+  if (isNaN(input.num2)) {
+    console.log(`${input.num2} is not a number`)
+    return
+  }
+  const num_2 = Number(input.num2)
 
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
+  switch (operation) {
+    case '+':
+    case 'add': 
+      console.log(`${num_1} + ${num_2} = ${num_1 + num_2}`);
+      break;
+    case '-':
+    case 'subtract':
+      console.log(`${num_1} - ${num_2} = ${num_1 - num_2}`);
+      break;
+    case '*':
+    case 'multiply':
+      console.log(`${num_1} * ${num_2} = ${num_1 * num_2}`);
+      break;
+    case '/':
+    case 'divide': 
+      if (num_2 === 0) {
+        console.log('You cannot divide by zero');
+        break;
+      }
+      console.log(`${num_1} / ${num_2} = ${num_1 / num_2}`);
+      break;
+    default:
+      console.log('unsupported operator');
+  }
```

Comment:
While you technically don't need to, it's a good habit to put a break even after the end of a `default` case!